extensions/ui-extension/src/targets/purchase/thank-you/block/render.tsx

```diff
@@ -1,14 +1,58 @@
 import { Community, Newsletter, GuideLinks } from '@/components/post-purchase';
 import { reactExtensionWithSentry } from '@/helpers/extension';
-import { BlockStack } from '@shopify/ui-extensions-react/checkout';
+import { isHeadlessStore } from '@/helpers/shop';
+import { useIfixitSettings } from '@/hooks/useIfixitSettings';
+import {
+   BlockStack,
+   useAttributes,
+   useShop,
+} from '@shopify/ui-extensions-react/checkout';
+import { Shop } from '@shopify/ui-extensions/checkout';
+import { useEffect } from 'react';
+import * as Sentry from '@sentry/browser';
 
 export default reactExtensionWithSentry(
    'purchase.thank-you.block.render',
-   () => (
+   () => <ThankYou />
+);
+
+function ThankYou() {
+   const attributes = useAttributes();
+   const publicCartid = attributes?.find(
+      (attr) => attr.key === 'publicCartid'
+   )?.value;
+   const shop = useShop();
+   const { ifixitBaseUrlDevOverride } = useIfixitSettings();
+   useEffect(() => {
+      void clearIfixitCart(publicCartid, shop, ifixitBaseUrlDevOverride);
+   }, [publicCartid, shop, ifixitBaseUrlDevOverride]);
+   return (
       <BlockStack>
          <Community />
          <Newsletter />
          <GuideLinks />
       </BlockStack>
-   )
-);
+   );
+}
+
+export const clearIfixitCart = async (
+   publicCartid: string | undefined,
+   shop: Shop,
+   ifixitBaseUrl: string | null
+) => {
+   if (publicCartid && ifixitBaseUrl && isHeadlessStore(shop)) {
```

Comment:
Non-blocking opinion: This could be a hook that doesn't accept any arguments. It also doesn't need to be exported.

```suggestion
function useClearIfixitCart() {
   const [publicCartid] = useAttributeValues(['publicCartid']);
   const ifixitBaseUrl = useIfixitBaseUrl();
   const shop = useShop();
   return async () => {
      if (publicCartid && ifixitBaseUrl && isHeadlessStore(shop)) {
```